feature/experimental/policy/policy_vrf_selection/ate_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go

```diff
@@ -473,6 +473,9 @@ func TestPBR(t *testing.T) {
 
 			//configure pbr policy-forwarding
 			gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).PolicyForwarding().Config(), tc.policy)
+			dutConfNIPath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance)
+			gnmi.Update(t, dut, dutConfNIPath.Type().Config(), oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
+
```

Comment:
Can you do this in the following order: default NI first, then update the PolicyForwarding

//configure pbr policy-forwarding
dutConfNIPath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance)
gnmi.Replace(t, dut, dutConfNIPath.Type().Config(), oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
gnmi.Update(t, dut, gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).PolicyForwarding().Config(), tc.policy)

Reply:
If you will do this, please rebase the PR as well.

Reply:
Updated the code to configure NI first followed by configure PolicyForwarding as suggested